src/utils/graph_building.jl

```diff
@@ -7,21 +7,23 @@ export inverse_square, exp_decay
 using PyCall
 using ChemistryFeaturization
 using Serialization
+using Xtals
+using NearestNeighbors
+rc[:paths][:crystals] = @__DIR__ # because Xtals has a silly default thing
```

Comment:
Can we specify here that we're doing this so Xtals knows to look for the crystal structure files in `@__DIR__`? Otherwise, we're not really specifying what the default is or why this is even required.

Reply:
```suggestion
rc[:paths][:crystals] = @__DIR__ # specify @__DIR__ as the directory where Xtals would look for the crystal structure files.
```

Reply:
I think I'm actually going to comment it out for now as @DhairyaLGandhi said that there are AD reasons why using `@__DIR__` is not good. If you don't set this it does still work, you just have to always feed it absolute paths.

Reply:
Relocatability with Package compiler reasons* :p